### DISCUSSION TITLE
Closed Caption menu (Video Subtitle's) in Maui - android platfrom

### DISCUSSION THREAD

**Discussion by Meysamsh1986** [CommunityToolkit/Maui#1704] [Q&A]
Hello, CommunityToolkit.Maui.MediaElement Closed Caption menu (Video subtitle's) is working fine in windows platform, but caption menu does not show in android platform, How can I fix this issue ?

**Comment by ghost** [comment]
@Meysamsh1986 if you could share you video url, I can try something as a DEMO project. [Mediaelement has been updated since Feb 2024](https://learn.microsoft.com/en-us/dotnet/communitytoolkit/maui/views/mediaelement?tabs=windows).
